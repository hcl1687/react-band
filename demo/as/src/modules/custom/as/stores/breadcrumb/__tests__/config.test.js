import configFactory from '../config'

const config = configFactory()

describe('custom/as/stores/breadcrumb/config', () => {
  it('should render correctly', () => {
    expect(config).toMatchSnapshot()
  })
})
