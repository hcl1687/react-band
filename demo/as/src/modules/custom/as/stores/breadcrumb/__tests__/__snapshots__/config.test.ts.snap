// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom/as/stores/breadcrumb/config should render correctly 1`] = `
Object {
  "name": "breadcrumbStore",
}
`;
